### PR TITLE
chore: update README

### PR DESCRIPTION
**Pull request by edvald**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,74 +1,115 @@
...
+```yaml
+kind: Module
+type: openfaas
+name: hello-function
+description: My OpenFaaS function
+lang: node
+```
+
+```yaml
+kind: Module
```
Comment (eysi09) [inline README.md]: This is a nitpick but could we stick to the

```yaml
kind
type
name
description
```
order that we use in the other example?

File: README.md [hunk 2]
```diff
@@ -1,74 +1,115 @@
...
+Garden is a _development orchestrator_ for Kubernetes, containers and functions, designed to make it easy to rapidly develop and test multi-service systems.
 
-- **Focus on what matters:** Garden continuously builds, tests and deploys your changes into your own persistent development environment as you code. Easily define and continuously run tests in the background, and focus on writing code.
+Using Garden, you can make your workflow reproducible and portable. With Garden, each part of your stack _describes itself_ using simple, intuitive declarations, without changing any code.
 
-- **Codify your workflow:** Make your workflow reproducible and portable. Define your stack using simple, intuitive declarations and get your environment up and running with a single command, without changing any code.
+Garden collects all of your declarations–even across multiple repositories—into a full graph of your stack, and leverages that information to **dramatically improve your developer experience**.
 
-![](docs/loop.gif)
+## Key features
```
Comment (eysi09) [inline README.md]: This feels a little long if you're just browsing more the most important / stand-out features.

Would it make sense to have a section where the two or three most important things about Garden jump out at you and then a more detailed list of features below?

**Comment by eysi09** [comment]
Do we also want to add a CTA for starring the repo?

File: README.md (hunk 2 above)
Comment (edvald) [reply README.md]: I think it'll make a little more sense like this once we add the diagram we talked about. But we can trim it down a bit. Thoughts on which parts feel to long or superfluous?

File: README.md [hunk 3]
```diff
@@ -1,74 +1,114 @@
...
+Garden collects all of your declarations–even across multiple repositories—into a full graph of your stack, and leverages that information to **dramatically improve your developer experience**.
 
-![](docs/loop.gif)
+> _If you’re using Garden or if you like the project, please ★ star this repository to show your support 💖_
 
-The project is in _alpha stage_. APIs may change, and platform stability and support is still limited. To understand the motivation behind Garden, [click here](https://garden.io/#motivation).
+## Key features
+
+- Spin up your whole stack with a single command, and (optionally) watch for changes. Because of the dependency graph, only what's needed gets re-built, re-deployed, and/or re-tested, so you get **much faster feedback loops**.
+- Easily write [integration test suites](https://docs.garden.io/using-garden/features-and-usage#testing-and-dependencies), that have runtime dependencies. Run tests before pushing your code to CI, and avoid having to mock or stub your own services.
```
Comment (thsig) [inline README.md]: ", that" -> "that"

File: README.md [hunk 4]
```diff
@@ -1,74 +1,114 @@
...
+- Define [tasks](https://github.com/garden-io/garden/tree/master/examples/tasks) that run as part of your deployment process, e.g. database migrations or scaffolding.
+- [Hot reload](https://docs.garden.io/using-garden/hot-reload) lets you near-instantaneously update code and static files in containers as they run, on services that support in-place reloading.
```
Comment (thsig) [inline README.md]: "on services" -> "for services"?

File: README.md [hunk 5]
```diff
@@ -1,74 +1,114 @@
...
+- [Remote sources](https://docs.garden.io/examples/remote-sources) support allows your project to automatically pull code from different repositories.
+- The built-in web **dashboard** gives you full overview of your stack (and many more UI features are planned to further aid with development).
```
Comment (thsig) [inline README.md]: "gives you full overview" -> "gives you a full overview"

and

"to further aid with" -> "to further aid"

**Pull request merged**